src/components/NcAppContent/NcAppContent.vue

```diff
@@ -268,6 +279,24 @@ export default {
 				},
 			}
 		},
+
+		realPageTitle() {
+			if (this.pageTitle) {
+				return this.pageTitle
+			}
+			if (this.pageHeading) {
+				return t('{view} - {app} - {product}', { view: this.pageHeading, app: appName, product: productName })
+			}
```

Comment:
Does it mean that for custom `pageTitle` the developer has to manually build string in a format of `{view} - {app} - {product}`?

Shouldn't we fix at least `{app} - {product}` or `{product}` parts, allowing to provide the view name? Or provide a layout like in [unhead](https://unhead.unjs.io/usage/guides/title-template)?

Do you have an example of expected `pageTitle` values?

For example, if this is for `Files` and I open `Foo` folder on `Favorites`, that `pageTitle` is expected?
1. `Favorites`
1. `Favorites - Nextcloud`
1. `Favorites - Files - Nextcloud`
1. `Foo`
1. `Foo - Files - Nextcloud`
1. `Foo - Favorites - Nextcloud`
1. `Foo - Favorites - Files - Nextcloud`

Reply:
The currently expected (but only implemented by a few apps) is `Favorites - Files - Nextcloud`.

Reply:
> The currently expected (but only implemented by a few apps) is `Favorites - Files - Nextcloud`.

Then isn't `pageTitle` used as a kind of second level heading like `pageHeading`?

`{pageTitle} - {pageHeading} - {productName}`

Reply:
We could also do it like that yes.

Maybe even standarize the visible headline like it is used on activity, so we could have the `h1` with the app name, the `h2` with the current view -> page title `view - app name - product name`.
But that would need more discussion.